src/components/ClockFace.jsx

```diff
@@ -29,23 +35,27 @@ const ClockFace = () => {
           } else {
             setCurrentSessionIndex(currentSessionIndex + 1);
           }
-          setCurrentSeconds(sessions[currentSessionIndex])
+          setCurrentSeconds(sessions[currentSessionIndex]);
           clearInterval(timeInterval);
         }
-      }, 1000)
+      }, 1000);
     } else if (!isActive) {
-      clearInterval(timeInterval)
+      clearInterval(timeInterval);
```

Comment:
I know you mentioned a new line for else if and else, but it comes from one of the linting extensions. I can remove the airbnb linting like you said for the next project if it is an overkill, and new line is a better practice.

Reply:
oh actually I prefer what you have right now. I don't prefer a one line `if` statement tho.
```js
if (x) return 'I do not like this'


if (x) {
  return 'I like this'
}
```